### ISSUE TITLE
Make Copilot Chat View searchable

### ISSUE THREAD

**Issue by Maddimax** [microsoft/vscode#259374]
Add a "find" function, same as in the text editor, so the user can search for text in the chat window

**Comment by yCodeTech** [comment]
Yeah, why isn't this added yet?

**Comment by CaliViking** [comment]
This feature would be highly valuable. The chat is increasingly becoming a workbook for the generation of code.
It is interesting how basic (and simple?) features like this are missing while more advanced (and complex?) features are implemented.

**Comment by tenequm** [comment]
Incredible... Such a basic feature...

I would even call it a bug, actually.

**Comment by Martin-Majernik** [comment]
Is there any update on this bug please ? Im having GitHub Copilot Pro version and it's really annoying that it doesn't have this basic feature. Also sine this demand has been created already more than two years ago :/ 
Is there any way we can help with that?

**Comment by felix-dm** [comment]
Bump 😜

**Comment by silpertan** [comment]
Lacking basic functionality might be forgivable if I was getting this for free, but lacking the ability to search my chat history seems self-serving - especially when I'm paying per-chat in many instances.

**Comment by JMLTT** [comment]
Mayor flaw that I cant search inside the copilot chat. It should not take years to implement.

**Comment by yCodeTech** [comment]
FINALLY! Copilot chat is now searchable in 1.134.0! It's just a shame that it took so long and that nobody keeps these issues informed of the development.

https://code.visualstudio.com/updates/v1_134#_find-in-chat

**Comment by adammw** [comment]
Seems it only searches within the current chat and still can't search all conversations (full text) yet?

**Comment by yCodeTech** [comment]
> Seems it only searches within the current chat and still can't search all conversations (full text) yet?

@adammw Yes it does only search the current chat, which is what a lot of users wanted anyway, and which I believe this issue is regarding. They may eventually add a search all chats/agents functionality, which would be the next logical step, seeing as though this new "find in (current) chat" functionality was only added presumably because of the Agents app development.
